spec/fixtures/mocks/submission_completed_no_business.json

```diff
@@ -9,7 +9,8 @@
       "order_count": 1,
       "purchase_order_number": null,
       "status": "completed",
-      "levy": 4500
+      "levy": 4500,
+      "report_no_business?": true
```

Comment:
:+1:

In this file, could you change `invoice_count` and `order_count` to `0`. As there wouldn't be any invoices or orders when reporting no business.